package.json

```diff
@@ -45,33 +45,32 @@
   },
   "homepage": "https://github.com/libp2p/js-libp2p-floodsub#readme",
   "devDependencies": {
-    "aegir": "^18.2.1",
+    "aegir": "^20.0.0",
     "benchmark": "^2.1.4",
     "chai": "^4.2.0",
     "chai-spies": "^1.0.0",
     "detect-node": "^2.0.4",
     "dirty-chai": "^2.0.1",
-    "libp2p": "~0.24.4",
+    "libp2p": "~0.26.1",
     "libp2p-secio": "~0.11.1",
     "libp2p-spdy": "~0.13.3",
-    "libp2p-tcp": "~0.13.0",
+    "libp2p-tcp": "~0.13.1",
     "libp2p-websocket-star": "~0.10.2",
-    "libp2p-websocket-star-rendezvous": "~0.3.0",
-    "lodash": "^4.17.11",
-    "multiaddr": "^6.0.6",
+    "libp2p-websocket-star-rendezvous": "~0.4.1",
+    "lodash": "^4.17.15",
+    "multiaddr": "^6.1.0",
     "peer-id": "~0.12.2",
     "peer-info": "~0.15.1",
-    "sinon": "^7.3.2"
+    "sinon": "^7.4.2"
   },
   "dependencies": {
     "async": "^2.6.2",
-    "bs58": "^4.0.1",
     "debug": "^4.1.1",
     "length-prefixed-stream": "^2.0.0",
-    "libp2p-crypto": "~0.16.1",
-    "libp2p-pubsub": "~0.2.0",
+    "libp2p-pubsub": "libp2p/js-libp2p-pubsub#refactor/async",
```

Comment:
The module is depending on itself?

Reply:
what do you mean by itself? It depends on [libp2p/js-libp2p-pubsub](https://github.com/libp2p/js-libp2p-pubsub), which is the base pubsub protocol where `floodsub` and `gossipsub` are built on top.

Reply:
I misread :)